pages/about-us/contact/en.md

```diff
@@ -1,47 +1,47 @@
 @title = 'Contact'
 
-h2. Help Tickets
+## Help Tickets
 
 The best way to get in touch with us is to fill out a help ticket. Please visit https://support.riseup.net to create a ticket. Note that you do not need to have an email account with riseup to file a help ticket.
 
-h2. Email
+## Email
 
 To contact us regarding general questions about our collective, send mail to collective@riseup.net. Any help questions directed to this email will be ignored.
 
-Our gpg long keyid is 0x3043E2B7139A768E. The full fingerprint is:
+Our gpg long keyid is `0x3043E2B7139A768E`. The full fingerprint is:
+
+	Key fingerprint = 4E07 9126 8F7C 67EA BE88  F1B0 3043 E2B7 139A 768E
 
-pre. Key fingerprint = 4E07 9126 8F7C 67EA BE88  F1B0 3043 E2B7 139A 768E
 
 There is no particular reason you should trust this key. You can see who trusts it:
 
-pre. gpg --list-sigs 0x4E0791268F7C67EABE88F1B03043E2B7139A768E
+	gpg --list-sigs 0x4E0791268F7C67EABE88F1B03043E2B7139A768E
+
 
-h2. Mail
+## Mail
 
 Our mailing address is:
 
-<pre>
-Riseup Networks
-PO Box 4282
-Seattle, WA 98194 USA
-</pre>
+	Riseup Networks
+	PO Box 4282
+	Seattle, WA 98194 USA
+
+
+## IRC
 
-h2. IRC
+	server : irc.indymedia.org
+	port for SSL connection : 6697
+	channel : #riseup
+	web interface : https://chat.indymedia.org
 
-<pre>
-server : irc.indymedia.org
-port for SSL connection : 6697
-channel : #riseup
-web interface : https://chat.indymedia.org
-</pre>
 
 (see [[chat/chat-security]] for information on keeping IRC secure)
 
-h2. Social Networks
+## Social Networks
 
 * Diaspora : "riseup@diasp.org":https://diasp.org/people/e6901810cb670133bdbb782bcb452bd5
 * Twitter : "@riseupnet":https://twitter.com/riseupnet
```

Comment:
The Diaspora and Twitter links don't render right in the converted version, instead they show the full link